tests/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -------------------------------------------------------------------------
```

Comment:
Do we intend to introduce a CI pipeline for all tests in this folder?
If these tests are specific to ort_moe, may we put all of them inside `ort_moe` folder?

Reply:
I tried moving tests files inside `ort_moe` folder but cannot fix import module path. So leaving it there now. Let's fix it in next refactoring